python/src/excalibur_detector/control/hl_detector.py

```diff
@@ -1524,7 +1524,8 @@ def power_card_read(self):
             # Check for the current HV enabled state
             hv_enabled = 0
             # Greater than hv_bias means the HV is enabled
-            if self._powercard_status['pwr_bias_vmon'] > self._hv_bias - 5.0:
```

Comment:
Not part of this fix but should we fix that magic number (5.0)?

Reply:
Is [this](https://github.com/dls-controls/excalibur-detector/blob/packaging/python/src/excalibur_detector/control/hl_detector.py#L1460) the same 5.0 or a different 5.0?